tools/NuGet/scripts/build_and_push_servicecore_debug.ps1

```diff
@@ -1,10 +1,10 @@
 # PowerShell script to build DynamoCore.sln in Debug, invoke BuildPackages.bat, and push the resulting nupkg to a local NuGet repo
 
 param(
-    [string]$SolutionPath = "..\..\..\src\DynamoCore.sln",
-    [string]$NugetScriptDir = "..\",
+    [string]$SolutionPath = "$PSScriptRoot\..\..\..\src\DynamoCore.sln",
```

Comment:
so does this always use the location of the script? neat.

Reply:
It does on 3.0 and later, so we should be ok.